aspnetcore/signalr/java-client.md

```diff
@@ -69,6 +69,19 @@ SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further detail
 
 This can safely be ignored.
 
+
+##Configuring Access Token Provider
```

Comment:
```suggestion
## Configure access token provider
```

Reply:
```suggestion
## Configure bearer token authentication
```

"Access Token Provider" is more of a means-to-an-end, so we should describe the thing the user wants to do.